src/main/java/bartworks/system/material/gtenhancement/PlatinumSludgeOverHaul.java

```diff
@@ -1053,25 +1054,19 @@ private static boolean isInBlackList(ItemStack stack) {
             }
         }
 
-        try {
-            if (Class.forName("gtPlusPlus.core.item.base.BaseItemComponent")
-                .isAssignableFrom(
-                    stack.getItem()
-                        .getClass())
-                && !stack.getUnlocalizedName()
-                    .contains("dust")
-                && !stack.getUnlocalizedName()
-                    .contains("Dust")) {
-                return true;
-            }
-            if (Class.forName("gtPlusPlus.core.block.base.BlockBaseModular")
-                .isAssignableFrom(
-                    Block.getBlockFromItem(stack.getItem())
-                        .getClass())) {
-                return true;
-            }
-        } catch (ClassNotFoundException e) {
-            e.printStackTrace();
+        if (BaseItemComponent.class.isAssignableFrom(
+            stack.getItem()
+                .getClass())
```

Comment:
I think `instanceof` can be used to improve readability:
```suggestion
        if (stack.getItem() instanceof BaseItemComponent
```